lib/dhttpd.dart

```diff
@@ -19,6 +20,8 @@ class Dhttpd {
 
   String get urlBase => 'http://$host:$port/';
 
+  static bool get isSSL => _ssl;
```

Comment:
Make this a normal final field and pass it in.

Reply:
Doh, yes sorry @kevmoo ! I did this is a bit of a rush and didnt properly review the code from the previous PR that I cherry picked.